codegen/service.go

```diff
@@ -82,10 +82,11 @@ func NewModuleSpec(
 	}
 
 	moduleSpec := &ModuleSpec{
-		WantAnnot:   wantAnnot,
-		IsEndpoint:  isEndpoint,
-		ThriftFile:  module.ThriftPath,
-		PackageName: packageName(module.GetName()),
+		CompiledModule: module,
```

Comment:
I leave this Upper case for now, `codegen_test` package will need to access, the test package there can be reorganize, maybe in a separate PR